packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/ar.ftl

```diff
@@ -0,0 +1,7 @@
+sync-reminder = هذا هو التذكير الخاص بك لمزامنة الأجهزة.
```

Comment:
Even this file is added for testing purpose. I'll remove it when the Pr is mergeable